docs/guide.rst

```diff
@@ -7,39 +7,47 @@ Basic experiment: get the data & crunch the stats
 
 Let's start by analysing a straightforward pref-flip experiment on the desktop browser.
 
-If we're using Databricks, we begin by installing the latest version of :mod:`mozanalysis` into the notebook (we don't install for the whole cluster, because then upgrades require restarting the entire cluster)::
+If we're using a Colab notebook, we begin by installing the latest version of :mod:`mozanalysis` into the notebook. It's a good idea to specify the specific version, for reproducibility::
 
-    dbutils.library.installPyPI("mozanalysis", "[current version]")
+    !pip install mozanalysis=='{current_version}'
 
-Then we import the necessary classes for getting the data, and for analysing the data::
+We take the per-notebook daily trudge::
+
+    from google.colab import auth
+    auth.authenticate_user()
+    print('Authenticated')
+
+Then we import the necessary classes for getting the data, and for analysing the data, and for interacting with BigQuery::
 
     import mozanalysis.metrics.desktop as mmd
     import mozanalysis.bayesian_stats.binary as mabsbin
     from mozanalysis.experiment import Experiment
+    from mozanalysis.bq import BigQueryContext
+
+
+And get a BigQuery context (a client, and some config)::
 
-For querying data, the general approach of :mod:`mozanalysis` is to start by obtaining a list of who was enrolled in what branch, when. This list is the ``enrollments`` Spark DataFrame; it has one row per client. Then we try to quantify what happened to each client: for a given analysis window (a specified period of time defined with respect to the client's experiment results.date), we seek to obtain a value for each client for each metric. So we end up with a results (pandas) DataFrame with one row per client and one column per metric.
+    bq_context = BigQueryContext(
+        dataset_id='{your_dataset_id}',  # e.g. mine's 'flawrence'
+    )
+
+If you're not part of ``moz-fx-data-bq-data-science``, then you'll also need to pass a ``project_id`` argument when initializing ``BigQueryContext()``.
```

Comment:
What does it mean to be part of moz-fx-data-bq-data-science? There's some kind of persistent default project somewhere, right?

Reply:
Oh, I see; "to query using a project other than..."